bp_common/test/src/bootrom/bootrom.S

```diff
@@ -1,3 +1,21 @@
+/*
+ * The bootrom loads the CCE ucode and the processor configuration and switches to the program for execution
+ */
+
+// CCE parameters
+#define CCE_PC_WIDTH 8
+#define NUM_CCE_INSTR (1 << CCE_PC_WIDTH)
+#define CCE_INSTR_END 0xFFFFFFFFFFFFFFFF
+
+// Configuration parameters
+#define CFG_CCE_UCODE_BASE_ADDR 0x00208000
+
+#define CFG_CCE_MODE_ADDR 0x00200081
+#define CFG_DCACHE_MODE_ADDR 0x00200043
+#define CFG_ICACHE_MODE_ADDR 0x00200022
+
+#define CFG_DOMAIN_MASK_ADDR 0x00200009
```

Comment:
Don't bother setting domain or sac masks here.  These are extremely rare conditions to actually change

Reply:
So, whats the alternative here? Leave them in the NBF script? Put them as part of the default_cfg?

Reply:
Don't touch them at all.  User software can deal with them if they need to